--- dbCmd.cpp	2016-10-10 23:18:53.000000000 +0800
+++ dbCmd.cpp.new	2016-10-10 23:08:07.000000000 +0800
@@ -96,7 +96,7 @@
    if (!CmdExec::lexSingleOption(option, token, false))
       return CMD_EXEC_ERROR;
    int c;
-   if (!checkColIdx(option, c)) return CMD_EXEC_ERROR;
+   if (!checkColIdx(token, c)) return CMD_EXEC_ERROR;
 
    float a = dbtbl.getAve(c);
    ios_base::fmtflags origFlags = cout.flags();
@@ -132,7 +132,7 @@
    if (!CmdExec::lexSingleOption(option, token, false))
       return CMD_EXEC_ERROR;
    int c;
-   if (!checkColIdx(option, c)) return CMD_EXEC_ERROR;
+   if (!checkColIdx(token, c)) return CMD_EXEC_ERROR;
 
    int n = dbtbl.getCount(c);
    cout << "The distinct count of column " << c << " is " << n << "." << endl;
@@ -208,7 +208,7 @@
    if (!CmdExec::lexSingleOption(option, token, false))
       return CMD_EXEC_ERROR;
    int c;
-   if (!checkColIdx(option, c)) return CMD_EXEC_ERROR;
+   if (!checkColIdx(token, c)) return CMD_EXEC_ERROR;
 
    float n = dbtbl.getMax(c);
    cout << "The max data of column " << c << " is " << n << "." << endl;
@@ -241,7 +241,7 @@
    if (!CmdExec::lexSingleOption(option, token, false))
       return CMD_EXEC_ERROR;
    int c;
-   if (!checkColIdx(option, c)) return CMD_EXEC_ERROR;
+   if (!checkColIdx(token, c)) return CMD_EXEC_ERROR;
 
    float n = dbtbl.getMin(c);
    cout << "The min data of column " << c << " is " << n << "." << endl;
@@ -279,7 +279,8 @@
 DBPrintCmd::usage(ostream& os) const
 {
    os << "DBPrint < (int rowIdx) (int colIdx)\n"
-      << "        | -Row (int rowIdx) | -Column (colIdx) | -Table>" << endl;
+      << "        | -Row (int rowIdx) | -Column (colIdx) | -Table | -Summary>"
+      << endl;
 }
 
 void
